Source/Logger/iOS/ios_logger.cpp

```diff
@@ -0,0 +1,8 @@
+#include "pch.h"
+
+#include <httpClient/trace.h>
+
+void Internal_HCTraceMessage(const char* areaName, HCTraceLevel traceLevel, const char* message)
```

Comment:
We should discuss if we want to move all the debug log formatting in this function as a future bit of work